### DISCUSSION TITLE
[@types/lodash] Missing pluck

### DISCUSSION THREAD

**Discussion by totszwai** [DefinitelyTyped/DefinitelyTyped#62825] [Issues with a @types package]
@types/lodash is missing `pluck` definition.

Is somewhere inside something call `fp` but is not being able to import it via:
```
import { pluck } from 'lodash';
```

We could see is somewhere in there...
```
node_modules/@types/lodash$ grep -r "pluck" *
fp/pluck.d.ts:import { pluck } from "../fp";
fp/pluck.d.ts:export = pluck;
fp.d.ts:        pluck: LodashMap;
```

<img width="365" alt="image" src="https://user-images.githubusercontent.com/4440365/197253422-9aa5928c-6609-4e8d-a3f5-06b400355143.png">

[a comment by an automated account is not shown]

**Reply by totszwai** [reply]
How do I close the discussion?

**Reply by Semigradsky** [reply]
You can'y close question. But you can answer.
